consensus/state_sync/module.go

```diff
@@ -117,13 +118,24 @@ func (m *stateSync) EnableServerMode() {
 // TODO(#352): Implement this function
 // Placeholder function
 func (m *stateSync) HandleGetBlockResponse(blockRes *typesCons.GetBlockResponse) error {
-	m.logger.Debug().Msgf("Received get block response: %s", blockRes.Block.String())
+	consensusMod := m.GetBus().GetConsensusModule()
+	serverNodePeerId := consensusMod.GetNodeAddress()
+	clientPeerId := blockRes.PeerAddress
+
+	m.logger.Info().Msgf("%s received get block response from: %s, for height %d. Received block's header is: %s,  \n", serverNodePeerId, clientPeerId, blockRes.Block.BlockHeader.Height, blockRes.Block.BlockHeader)
```

Comment:
1. try to use `GetGlockResponse` (the full proto, type or var name) where appropriate. It'll make it much easier to read in the logs.
2. You have lots of  parameters here (height, heater, etc...). Look at how you can set fields: https://github.com/pokt-network/pocket/wiki/Logger-Readme#fields

Reply:
thanks for the suggestions!